x-pack/plugins/actions/server/builtin_action_types/alert_history_es_index/alert_history_es_index.ts

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
I wonder if `builtin_action_types/alert_history_es_index/*` is the best place for these preconfigured connectors or if a new folder like `preconfigured_connectors/alert_history_es_index/*` would be better?

Reply:
I never look at the directory names in these PRs :-)  good catch.

I think it probably should **NOT** be in `builtin_action_types/alert_history_es_index`, since it's actually not a new action type, it's just a customized index action with a bunch of extra setup-time stuff.

I'd be tempted to put the files `builtin_action_types/lib`, but that doesn't feel quite right either - another subdirectory in that directory named `alert_history_.es_index`  seems too deep / hard to find ... so Mike's suggestion of a new `preconfigured_connectors` directory, as a peer of `builtin_action_types`, seems like a good idea.

Alternatively, we could create a new directory for just the `es_index` connector (like `jira`, `resilient`, etc) and move everything under there.

Reply:
Updated in https://github.com/elastic/kibana/pull/94909/commits/75dcb88471d459ac14c4332bb90c4930319663d4 to `preconfigured_connectors/alert_history_es_index/*`